safe/test/utilities.py

```diff
@@ -232,7 +232,7 @@ def standard_data_path(*args):
     :param args: List of path e.g. ['control', 'files',
         'test-error-message.txt'] or ['control', 'scenarios'] to get the path
         to scenarios dir.
-    :type args: list
+    :type args: str
```

Comment:
should be: list[str]
because args is a list of string from the looks of it in the code.